services/ProductService.js

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios';
+
+const apiUrl = 'https://panda-market-api-crud.vercel.app';
+
+async function getProductList(page, pageSize, keyword) {
+    try {
+        const res = await axios.get(apiUrl+'/products', {
+            page,
+            pageSize,
+            keyword
+        });
```

Comment:
axios에서 쿼리 파라미터들은 params로 넣어야 동작합니다.
```suggestion
  const res = await axios.get(apiUrl+'/products', {
      params: {
          page,
          pageSize,
          keyword
      }
  });
```